src/bundle/Controller/ContentTypeController.php

```diff
@@ -692,6 +692,18 @@ public function viewAction(
         ContentType $contentType,
         Request $request
     ): Response {
+        $contentTypeGroups = $contentType->getContentTypeGroups();
+        $contentTypeGroupsIds = array_column($contentTypeGroups, 'id');
+        if (!in_array($group->id, $contentTypeGroupsIds, true)) {
+            throw $this->createNotFoundException(
+                sprintf(
+                    '%s Content Type does not belong to %s Content Type Group.',
```

Comment:
probably
```suggestion
                    '%s content type does not belong to %s content type group.',
```
after microcopy standards changes last year.

Reply:
I would suggest:
```suggestion
                    '%s Content type does not belong to %s content type group.',
```
since this is a sentence.

Reply:
Resolved